Dockerfile

```diff
@@ -0,0 +1,25 @@
+# distroless is debian based and has python 3.7.3
+# 3-alpine is 3.9.6
+FROM python:3.7-alpine AS build
+ARG POETRY_VERSION=1.1.7
+#ARG POETRY_HASH=d9ece46b4874e93e1464025891d14c83d75420feec8621a680be14376e77198b
+RUN apk add --no-cache build-base libffi-dev rust cargo openssl-dev && \
```

Comment:
Do we need all these dependencies? I would expect that every library is available as a wheel.

Reply:
Without installing these dependencies `poetry` won't install properly. It's installer doesn't get an error but if you try to run it, it won't work. If you look at the installer logs you can see its failing due to missing dependencies. These are build-time dependencies and do not end up in the final image.

PyPI only recently approved https://www.python.org/dev/peps/pep-0656/ to add support for `musl` lib-c, which Alpine Linux is based off. Over time more python packages should have musl-based wheels available.

If you prefer the build image could be based off `debian:buster-slim`. From my testing it will need `python3-venv` and `wget` installed via `apt-get`. Then `python`, `pip`, `poetry`, and `s3-account-search` will work.

I haven't benchmarked final image size nor build-time.

Reply:
It was mostly me being curious :)

Reply:
Thanks for merging. I was too slow in updating this PR. I've opened #3 that has a faster build time for the initial stage. It's glibc based, matching distroless.